wasm2c/README.md

```diff
@@ -342,32 +316,32 @@ The first few hundred lines define macros that are used to implement the
 various WebAssembly instructions. Their implementations may be interesting to
 the curious reader, but are out of scope for this document.
 
-Following those definitions are various initialization functions (`init`,
+Following those definitions are various initialization functions (`init`, `free`,
 `init_func_types`, `init_globals`, `init_memory`, `init_table`, and
 `init_exports`.) In our example, most of these functions are empty, since the
 module doesn't use any globals, memory or tables.
 
 The most interesting part is the definition of the function `fac`:
 
 ```c
-static u32 fac(u32 p0) {
+static u32 w2c_fac(u32 w2c_p0) {
   FUNC_PROLOGUE;
-  u32 i0, i1, i2;
-  i0 = p0;
-  i1 = 0u;
-  i0 = i0 == i1;
-  if (i0) {
-    i0 = 1u;
+  u32 w2c_i0, w2c_i1, w2c_i2;
+  w2c_i0 = w2c_p0;
+  w2c_i1 = 0u;
+  w2c_i0 = w2c_i0 == w2c_i1;
+  if (w2c_i0) {
+    w2c_i0 = 1u;
   } else {
-    i0 = p0;
-    i1 = p0;
-    i2 = 1u;
-    i1 -= i2;
-    i1 = fac(i1);
-    i0 *= i1;
+    w2c_i0 = w2c_p0;
+    w2c_i1 = w2c_p0;
+    w2c_i2 = 1u;
+    w2c_i1 -= w2c_i2;
+    w2c_i1 = w2c_fac(w2c_i1);
+    w2c_i0 *= w2c_i1;
   }
   FUNC_EPILOGUE;
-  return i0;
+  return w2c_i0;
```

Comment:
Do these change reflect the current output?  i.e. do we prefix all locals with w2c?

Reply:
I believe so -- I just copied and pasted from the current generated `fac.c`.